deploy/gke/private.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project: datcom-mixer-private
+ip: 34.117.109.110
+domain: mixer.endpoints.datcom-mixer-private.cloud.goog
+api_title: DataCommons API (Private)
+region: us-central1
+nodes: 1
+store:
+bucket: datcom-public
```

Comment:
Do you mean the bucket to be called datcom-public in a private instance?

Reply:
Good point, I have been putting the test files in this bucket. Can create a dedicated bucket later for this instance.